_data/toc.yaml

```
@@ -1439,12 +1439,12 @@ manuals:
   - sectiontitle: Install Compose
     section:
       - path: /compose/install/
-        title: Install Docker Compose 
-      - path: /compose/install/compose-desktop/
-        title: Install Compose plugin through Desktop 
-      - path: /compose/install/compose-plugin/
-        title: Install Docker Compose Plugin 
-      - path: /compose/install/uninstall/ 
+        title: Install Docker Compose
+      - path: /compose/install/compose-linux/
+        title: Install on Linux
+      - path: /compose/install/compose-other/
+        title: Other scenarios install
```

To me, "Other install scenarios" reads a bit more familiar than "Other scenarios install"

```suggestion
        title: Other install scenarios
```